crux-http-server/src/crux/http_server.clj

```diff
@@ -321,9 +321,11 @@
               :or {server-port 3000 cors-access-control []}
               :as options}]
   (s/assert ::options options)
-  (let [server (j/run-jetty (-> (partial handler crux-node)
-                                (#(apply wrap-cors (into [%] cors-access-control)))
+  (let [wrap-cors' #(apply wrap-cors (cons % cors-access-control))
+        server (j/run-jetty (-> (partial handler crux-node)
+                                (wrap-exception-handling)
                                 (p/wrap-params)
+                                (wrap-cors')
                                 (wrap-exception-handling))
```

Comment:
This is a double duct-tape but not a strong opinion here

Reply:
hmm, should have one at the end no?

Reply:
I added `(wrap-exception-handling)` before `(wrap-cors')`, otherwise exception responses don't get `Access-Control-Allow-Origin` header and get screwed by browser.